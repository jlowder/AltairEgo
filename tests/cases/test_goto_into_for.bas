10 PRINT "TEST: GOTO INTO FOR LOOP"
20 GOTO 40
30 FOR I = 1 TO 5
40 PRINT "INSIDE LOOP"
50 NEXT I
60 PRINT "TEST FAILED"
70 END
