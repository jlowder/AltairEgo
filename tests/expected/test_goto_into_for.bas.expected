Altair Ego: Emulating Altair BASIC 32K Rev. 3.2
OK
OK
TEST: GOTO INTO FOR LOOP
INSIDE LOOP
NEXT WITHOUT FOR
OK
